client/src/components/Chat/DropDownOption.jsx

```diff
@@ -12,9 +12,9 @@ import { socket } from 'src/lib/socketConnection';
 
 import useChatUtils from 'src/lib/chatSocket';
 
-const DropDownOptions = ({ id, isSender, inputRef, cancelEdit, setEditing, setReplyId }) => {
+const DropDownOptions = ({ id, isSender, inputRef, cancelEdit, setEditing, setReplyId ,cryptoKey,importedPrivateKey}) => {
 	const { app } = useApp();
-
+  
 	const { messages: state, updateMessage, removeMessage } = useChat();
```

Comment:
So that means you use the hook hear and remove the passing down

Reply:
In DropDownOption.jsx, when I use the hook useCryptoKeys, the values importedPrivateKey and cryptoKey are showing as null
![whispercopy02](https://github.com/user-attachments/assets/5e2e86be-aedd-412c-8429-d85df6f383a8)
![whispercopy03](https://github.com/user-attachments/assets/18bff372-5fc5-411b-a207-a1c577bcdfa1)

Reply:
"The importedPublicKey, importedPrivateKey, and cryptoKey are all null because the hook is not implemented correctly. If you give me permission, I can resolve this issue. ![whispergenerate](https://github.com/user-attachments/assets/40f1b248-1a0d-4cef-a930-98265e197936)
In the chat.jsx file, the keys are retrieved because the generateKeyPair() function is available. If generateKeyPair() is not available, the keys cannot be retrieved.
![whispergenerate02](https://github.com/user-attachments/assets/33586043-dd90-4776-bcfe-66f070107000)

Reply:
Sure